python-squid/src/main/java/org/sonar/python/api/tree/PyCallExpressionTree.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.api.tree;
+
+import com.sonar.sslr.api.Token;
+import java.util.List;
+
+public interface PyCallExpressionTree extends PyExpressionTree {
+  PyExpressionTree callee();
+
+  List<PyArgumentTree> arguments();
```

Comment:
Another model would have been to define a `PyArgumentListTree`. I think it would have represented [python calls](https://docs.python.org/3/reference/expressions.html#calls) more precisely. However, I think it would have required changed in the grammar.